config.template.yml

```diff
@@ -4,26 +4,29 @@
 #                           Authelia Configuration                            #
 ###############################################################################
 
-## The host and port to listen on.
```

Comment:
Not sure why this change has been done in this PR but for next time I would advise to do that in a dedicated PR instead. It would help keep the review focused, make it simpler for the reviewer and also it's better to track changes in the history since we squash commits.

Reply:
This was initially done to avoid a k8s conflict. The question is if we still want to do it if we go with the double underscore instead of single underscore.